diff --git a/dist/pipeline.js b/dist/pipeline.js
index 87354d6a2ab51c086f2543b6bf4e6bd781420e14..bcdc0294d0c9e20ef8453848b5b1984212420888 100644
--- a/dist/pipeline.js
+++ b/dist/pipeline.js
@@ -30,7 +30,7 @@ var re = (e) => e.replace(/\W/g, ""), ie = (e, t, n, r) => {
 	",",
 	"\r",
 	"\n"
-], ce = (e) => se.some((t) => e.includes(t)), B = (e) => `"${e.replace(/"/g, "\"\"")}"`;
+], ce = (e) => se.some((t) => e.includes(t)) || /^[=+\-@\t\r]/.test(e), B = (e) => `"${(/^[=+\-@\t\r]/.test(e)?"'":"")+e.replace(/"/g, "\"\"")}"`;
 function V(e) {
 	if (e == null) return e;
 	if (typeof e == "object") {
